backend/audit/views.py

```diff
@@ -598,6 +602,9 @@ def post(self, request, *args, **kwargs):
                 return render(request, "audit/upload-report.html", context)
         except SingleAuditChecklist.DoesNotExist:
             raise PermissionDenied("You do not have access to this audit.")
-        except Exception as e:
-            logger.info("Unexpected error in UploadReportView post.\n", e)
-            raise BadRequest()
+        except LateChangeError:
+            return render(request, "audit/already-certified.html")
```

Comment:
Nitpick: I wonder if we should call this something other than "already certified", since "ready for certification" (but not actually certified) would also fall into this late change category

Reply:
I changed this to `no-late-changes.html`